wordpress-mobile/WordPress-iOS
# New Jetpack prologue screen - prompts with animation

Ref #19228, #19229

This PR implements the new Jetpack prologue (landing) screen, with a series of animated prompts that rotate vertically (see animations below)

iPhone dark | iPad light
-- | --
<img height=600 src=https://user-images.githubusercontent.com/34376330/187795826-bb9b42ae-7668-447f-8235-9c8fc3aea0fc.gif> | <img height=600 src=https://user-images.githubusercontent.com/34376330/187795842-96d253c8-c313-4ae1-bebd-4d28603fe3fa.gif>

**KNOWN ISSUES:**

**This PR is intended to be the first step that implements this view. More PRs will come to address the issues below:**

- It's not optimized for iPad landscape
- The animation is not completely smooth (although it's smoother than the animated gifs show): we might need to tweak some parameters
- The gradient it's not quite right: I tweaked the existing one, but needs more work: one thing that's noticeable is that it does not adapt to interface style changes (from light to dark and vice versa)
- There is no blur effect / vibrancy (as was shown in the sample animations)

To test:

- checkout this branch and set the `newLandingScreen` feature flag to true
- Build the Jetpack target, and log out if you're logged in
- Observe the prologue screen and make sure it matches the above animations

## Regression Notes
1. Potential unintended areas of impact
None, it's a new set of classes, feature flagged for now

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
